Gruntfile.coffee

```diff
@@ -13,11 +13,20 @@ module.exports = (grunt) ->
   grunt.initConfig
     env: ENV
     pkg: grunt.file.readJSON('package.json')
+
+    flavors: [
+      'skroutz'
+      'alve'
+      'scrooge'
+    ]
+
     clean:
       payload:
-        src: ['dist/js/payload*']
+        src: ["dist/<%= grunt.config.get('current_flavor') %>/js/payload*"]
       plugins:
-        src: ['dist/js/plugins/*']
+        src: ["dist/<%= grunt.config.get('current_flavor') %>/js/plugins/*"]
+      samples:
+        src: ['src/settings.coffee.sample.1', 'src/plugins_settings.coffee.sample.1']
```

Comment:
I've added a preprocessing step for the samples, where `@@flavor` is replaced with the flavor being built. So:
```
@@@@flavor.analytics_base_url
```
becomes:
```
@@skroutz.analytics_base_url
```
The latter is finally replaced depending on the configuration in the `.yml` file:
```
skroutz:
  analytics_base_url: "https://analytics.skroutz.gr"
  application_base_url: "https://skroutz.gr"
```